Mathlib/Algebra/Group/Opposite.lean

```diff
@@ -410,8 +410,8 @@ theorem Units.coe_opEquiv_symm {M} [Monoid M] (u : Mˣᵐᵒᵖ) :
 /-- A semigroup homomorphism `M →ₙ* N` can equivalently be viewed as a semigroup homomorphism
 `Mᵐᵒᵖ →ₙ* Nᵐᵒᵖ`. This is the action of the (fully faithful) `ᵐᵒᵖ`-functor on morphisms. -/
 @[to_additive
-      "An additive semigroup homomorphism `add_hom M N` can equivalently be viewed as an\n
-      additive semigroup homomorphism `add_hom Mᵃᵒᵖ Nᵃᵒᵖ`. This is the action of the\n
```

Comment:
Are you sure these `\n`s should be removed?

Reply:
I think so. The `\n`s were added in by mathport. See the (original mathlib3 file)[https://github.com/leanprover-community/mathlib/blob/master/src/algebra/group/opposite.lean], where new lines are inserted simply to keep the lines to within 100 characters long.